packetbeat/tests/system/test_0002_thrift_basics.py

```diff
@@ -7,13 +7,13 @@ def tutorial_asserts(self, objs):
         assert len(objs) == 17
         assert all([o["type"] == "thrift" for o in objs])
 
-        # bytes_in present for all. bytes_out present for
+        # source.bytes present for all. destination.bytes present for
         # all except the zip async calls.
-        assert all([o["bytes_in"] > 0 for o in objs])
-        assert all([o["bytes_out"] > 0 for o in objs[0:14]])
-        assert all([o["bytes_out"] > 0 for o in objs[16:]])
-        assert objs[14]["bytes_out"] == 0
-        assert objs[15]["bytes_out"] == 0
+        assert all([o["source.bytes"] > 0 for o in objs])
+        assert all([o["destination.bytes"] > 0 for o in objs[0:14]])
+        assert all([o["destination.bytes"] > 0 for o in objs[16:]])
+        #assert objs[14]["destination.bytes"] == 0
```

Comment:
Commeted out on purpose? Also 2 more below.

Reply:
Zero values are automatically omitted based on how I marshaling these fields. Effectively it's like `json:"destination.bytes,omitempty"`. I meant to remove these lines. Will fix.